app/app.go

```diff
@@ -1017,34 +1061,37 @@ func New(
 	app.SetBeginBlocker(app.BeginBlocker)
```

Comment:
Probably need SetPreBlocker as well "When using (legacy) application wiring, the following must be added to app.go".

`app.SetPreBlocker(app.PreBlocker)`

Reply:
I've got it in the list of TODOs in the [epic](https://github.com/provenance-io/provenance/issues/1760).